taxonomies/categories.txt

```diff
@@ -39205,11 +39205,11 @@ zh:水果
 wikidata:en:Q1364
 
 <en:Fruits
-en:French fruits
+en:Fresh fruits
+fr:Fruits frais
 
 <en:Fruits
-fr:Cassis frais
-season_in_country_fr:en: 6,7,8
+en:French fruits
```

Comment:
Why do we have a "French fruits" category?

Reply:
Pierre created it for the DOP fruits I think